Profiles/cio-Reaction.xml

```diff
@@ -15,7 +15,7 @@
       <use value="work" />
     </telecom>
   </contact>
-  <description value="The diagnostic insight that the patient has developed a reaction as a result of exposure to a specific substance or group of substances." />
+  <description value="An adverse clinical response that is possible, probable or proven to result from exposure to a substance, group of substances or radiation." />
```

Comment:
Change this to ' ... to a substance or group of substances.', even though the data set still mentions radiation. Since this FHIR profile at this moment does not make it possible to specify reactions related to radiation, it's better to remove this text. Moreover, the AllergyIntolerance resource is **not** suited to convey information regarding reactions on radiation. For this the Condition resource is needed.